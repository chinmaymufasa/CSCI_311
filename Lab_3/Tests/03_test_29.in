2
88
5
2
109
5
2
118
5
2
143
5
2
98
5
2
149
5
2
158
5
2
109
5
2
28
5
2
96
5
2
34
5
2
86
5
2
136
5
2
20
5
2
153
5
2
131
5
2
127
5
2
119
5
2
166
5
2
139
5
1
20
-1