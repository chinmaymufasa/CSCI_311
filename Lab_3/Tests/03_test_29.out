88 
88 109 
88 109 118 
88 109 118 143 
88 98 109 118 143 
88 98 109 118 143 149 
88 98 109 118 143 149 158 
88 98 109 109 118 143 149 158 
28 88 98 109 109 118 143 149 158 
28 88 96 98 109 109 118 143 149 158 
28 34 88 96 98 109 109 118 143 149 158 
28 34 86 88 96 98 109 109 118 143 149 158 
28 34 86 88 96 98 109 109 118 136 143 149 158 
20 28 34 86 88 96 98 109 109 118 136 143 149 158 
20 28 34 86 88 96 98 109 109 118 136 143 149 153 158 
20 28 34 86 88 96 98 109 109 118 131 136 143 149 153 158 
20 28 34 86 88 96 98 109 109 118 127 131 136 143 149 153 158 
20 28 34 86 88 96 98 109 109 118 119 127 131 136 143 149 153 158 
20 28 34 86 88 96 98 109 109 118 119 127 131 136 143 149 153 158 166 
20 28 34 86 88 96 98 109 109 118 119 127 131 136 139 143 149 153 158 166 
20
